compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -413,6 +411,8 @@ void addGPUTileAndFusePassPipeline(OpPassManager &funcPassManager,
     funcPassManager.addPass(createGPUApplyTilingLevelPass(options));
   }
   funcPassManager.addPass(IREE::GPU::createDistributeMmaToLanesPass());
+  funcPassManager.addPass(createConvertToDestinationPassingStylePass(
```

Comment:
This pass is not really meant to be used here. I don't know what specifically from this is being relied upon. Id rather not use this pass directly cause it pulls in way more stuff than is needed for this use I believe.

Reply:
it might not be needed anymore. I've just been rebasing periodically and will see if it can be dropped now.